Tasks/UseDotNetV2/task.json

```diff
@@ -37,20 +37,38 @@
             "type": "pickList",
             "label": "Package to install",
             "defaultValue": "sdk",
-            "required": false,
+            "required": true,
```

Comment:
a pickList can't be empty. So it is always required.  The user hasn't a option to leave this empty because the UI disallows it. So it is implicitly required. So I can reset this value but it wouldn't change anything. 🤔

Reply:
And in the code behind there is also an implicit (intransparent) requirement of this parameter. Because if the user leave this empty (I don't know how the user can set the value to empty...) the code in usedotnet.ts will set it to `sdk`.

Reply:
User can create a pipeline usibg yml file, where they can choose not to enter the value.

Reply:
Oh I see... 

But then I have a question. The input definition defines a default value. Is this not use in the `yml` process?

Reply:
I have fix this. :)